Kickstarter-iOS/Views/Controllers/PledgeContinueViewController.swift

```diff
@@ -70,12 +70,10 @@ final class PledgeContinueViewController: UIViewController {
 
   private func goToLoginSignup(with intent: LoginIntent) {
     let loginSignupViewController = LoginToutViewController.configuredWith(loginIntent: intent)
+
     let navigationController = UINavigationController(rootViewController: loginSignupViewController)
-    let sheetOverlayViewController = SheetOverlayViewController(
-      child: navigationController,
-      offset: Layout.Sheet.offset
```

Comment:
I realized after doing a bunch of testing that we can't really use the same offset for every usage of the sheet overlay. This offset was way too big for the login modal, and it's also too big for landscape mode. We should address this issue separately, but for the shipping selection modal we'll need to be a bit smarter about how we define the offset.